src/frontend/Komponenter/Behandling/Inngangsvilkår/Inngangsvilkår.tsx

```diff
@@ -41,9 +44,17 @@ const Inngangsvilkår: FC<Props> = ({ behandlingId }) => {
         <DataViewer response={{ vilkår, behandling }}>
             {({ vilkår, behandling }) => {
                 const skalViseSøknadsdata = behandling.behandlingsårsak === Behandlingsårsak.SØKNAD;
+                const grunnlagsdataInnhentetDato = formaterIsoDatoMedTid(vilkår.opprettetTid);
 
                 return (
                     <>
+                        <OppdaterOpplysninger
+                            visningstekst={'Opplysninger hentet fra Folkeregisteret'}
```

Comment:
Burde visningstekst kunne inlines? (flyttes til `OppdaterOpplysninger`)

Reply:
Nei, det er nok ikke behov for inline-tekst, så flytter den inn i komponenten. Fikset i 107ba7e